react/react1/week1/Todo_list/src/App.jsx

```diff
@@ -0,0 +1,34 @@
+import './App.css'
+
+const App = () => {
+  const todos = [
+    { id: 1, description: 'Get out of bed', when: 'Wed Sep 13 2017' },
+    { id: 2, description: 'Brush teeth', when: 'Thu Sep 14 2017' },
+    { id: 3, description: 'Eat breakfast', when: 'Fri Sep 15 2017' },
+  ]
+
+  return (
+    <div>
+      <h2>Todo List</h2>
+      {todos.map((todo) => (
+        <TodoItem
+          id={todo.id}
```

Comment:
Doesn't seem like you use id in the todo item, so this could have been left out.

Reply:
I know, but I thought that using id in arrays is a rule of etiquette ))

Reply:
I understand, thanks!